### PR TITLE
fix(products): remove use of Products service

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

e15de3c - fix(deps): add ovh-api-services peer dependency
c7f1fff - fix(products): remove use of Products service

ref: OM-185

### Related PRs

- [x] https://github.com/ovh-ux/ovh-api-services/pull/113

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

**Comment by antleblanc** [comment]
[`ovh-api-services`](https://github.com/ovh-ux/ovh-api-services/releases/tag/v3.22.0) has been released `v3.22.0`.

File: src/exchange/resource/resource.service.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 angular.module('Module.exchange.services').service(
   'ExchangeResources',
   class ExchangeResources {
-    constructor($rootScope, Products, $http, $q, Exchange, OvhHttp) {
+    constructor($rootScope, $http, $q, Exchange, OvhHttp) {
```
Comment (marie-j) [inline src/exchange/resource/resource.service.js]: As they are not too many injections, maybe they could have been sorted ?

File: src/exchange/exchange.service.js [hunk 2]
```diff
@@ -199,7 +225,10 @@ angular.module('Module.exchange.services').service(
         this.resetCache();
       }
 
-      return this.services.Products.getSelectedProduct(forceRefresh)
+      const { organization, productId } = this.getParams();
+
+      // return this.services.$q.when({name: productId, organization})
```
Comment (FredericEspiau) [inline src/exchange/exchange.service.js]:
```suggestion
```

Comment (cbourgois) [reply src/exchange/exchange.service.js]: For sure ;)

File: src/exchange/resource/resource.service.js (hunk 1 above)
Comment (cbourgois) [reply src/exchange/resource/resource.service.js]: Done

**Comment by cbourgois** [comment]
`ovh-api-services@^3.22.0` added in peerDependencies.

**Pull request merged**